pkg/kubelet/kubelet_network_linux.go

```diff
@@ -37,7 +37,7 @@ const (
 )
 
 func (kl *Kubelet) initNetworkUtil() {
-	iptClients := utiliptables.NewDualStack()
+	iptClients, _ := utiliptables.NewDualStack()
 	if len(iptClients) == 0 {
```

Comment:
can this please be changed to:

```suggestion
	if err != nil || len(iptClients) == 0 {
```

And `err` added to the log.

Relying on return value when the err is not nil is not a good practice